package.json

```diff
@@ -30,6 +30,31 @@
     "mainClass": "XCUITestDriver",
     "scripts": {
       "build-wda": "./build/lib/build-wda.js"
+    },
+    "schema": {
+      "$schema": "http://json-schema.org/draft-07/schema",
+      "type": "object",
+      "properties": {
+        "webdriveragent-port": {
+          "appiumCliDest": "wdaLocalPort",
+          "default": 8100,
+          "description": "Local port used for communication with WebDriverAgent",
+          "maximum": 65535,
+          "minimum": 1,
+          "type": "integer"
+        },
+        "webkit-debug-proxy-port": {
```

Comment:
I remember when we want to mark some argument as deprecated, do we have action or something like https://github.com/appium/appium/blob/1e30207ec4e413c64396420fbb0388392e88cc54/lib/parser.js#L404 ?
(Or other place? sorry if I missed relevant discussion)

Reply:
There's currently no support for deprecated arguments, since we've dropped all of them.  It can trivially be added